bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/service.js

```diff
@@ -35,19 +36,19 @@ let getSlideData = (params) => {
 
 const previousSlide = (currentSlideNum) => {
   if (currentSlideNum > 1) {
-    makeCall('switchSlideMessage', currentSlideNum - 1);
+    makeCall('switchSlideMessage', currentSlideNum - 1,Auth.credentials);
```

Comment:
All the makeCall calls go to a function where Auth.credentials is passed on. (it's refactored out) Why are you passing Auth.credentials from here (in lots of places)?